src/main/kotlin/no/nav/arbeidsgiver/tiltakrefusjon/refusjon/Tiltakstype.kt

```diff
@@ -1,5 +1,7 @@
 package no.nav.arbeidsgiver.tiltakrefusjon.refusjon
 
 enum class Tiltakstype {
-    MIDLERTIDIG_LONNSTILSKUDD, VARIG_LONNSTILSKUDD, SOMMERJOBB, VTAO
-}
\ No newline at end of file
+    MIDLERTIDIG_LONNSTILSKUDD, VARIG_LONNSTILSKUDD, SOMMERJOBB, VTAO;
+
+    fun harFastUtbetaling() = this == VTAO
```

Comment:
```suggestion
    fun utbetalesAutomatisk() = this == VTAO
```

ellernoe sånt kanskje? Synes kanskje harFastUtbetaling er litt vanskelig å skjønne hva betyr. De fleste steder denne funksjonen sjekkes bruker man den også for å sjekke om det er automatisk utbetaling.

Kunne det også vært en property på enumene? Typ. utbetalesAutomatisk: boolean?

Reply:
Støtter nytt navn! https://github.com/navikt/tiltak-refusjon-api/pull/130/commits/13e9ba705e7ef06e33d55483f62b14b82493a432